users/script.js

```diff
@@ -328,22 +329,24 @@ function clearFilters() {
   );
 }
 
-async function showUserDataList(
+const showUserDataList = async (
```

Comment:
Why you are changing function declaration to arrow function?

Reply:
The change in function declaration from function expression to arrow function was made to maintain consistency in coding style and improve readability. Arrow functions provide a more concise syntax and lexical scoping, which can be beneficial in certain contexts. If you prefer the original function declaration, I can revert it.